modules/slf4j/src/main/java/org/apache/ignite/logger/slf4j/Slf4jLogger.java

```diff
@@ -65,7 +66,12 @@ public Slf4jLogger(Logger impl) {
 
         this.impl = impl;
 
-        quiet = Boolean.valueOf(System.getProperty(IGNITE_QUIET, "true"));
+        quiet = Boolean.parseBoolean(System.getProperty(IGNITE_QUIET, "true"));
+
+        // Optionally remove existing handlers attached to j.u.l root logger (since SLF4J 1.6.5)
+        SLF4JBridgeHandler.removeHandlersForRootLogger();
```

Comment:
What does this thing do? Is it required?
The comment looks confusing. It says "Optionally", meaning that this step is not really required. We decided to perform it, but kept the "Optionally" word.

Reply:
Remove all other handlers that can catch logs from JUL. I think it required.
Remove "Optionally" word.

Reply:
Let's remove the "(since SLF4J 1.6.5)" part as well then.